ui/src/index.ts

```diff
@@ -1,7 +1,3 @@
+export * from '@neighbourhoods/provider-store'
 export * from './contexts'
```

Comment:
We no longer need to export a manifest of all modules since each module is able to be included by its own package name.

Reply:
What's exported here seems to be the minimum API footprint for a We applet. Things seem to be naturally falling into their correct places when it comes to area of responsibility: everything to do with the Provider is now contained within `@neighbourhoods/provider-store`, and `@neighbourhoods/provider-applet` only exports its root `ProviderApp` and all relevant contexts & stores needed for initialisation.

Probably `sensemakerStoreContext` could be defined in `@neighbourhoods/nh-we-applet` rather than in `@neighbourhoods/provider-applet` but I can also see how multiple sensemaker contexts for different apps with different IDs could be desirable.

Reply:
One other mention: if we also want to have a root package manifest that can act as a single module for import into other applications then, yes, it's very easy to re-export all the Applet's UI modules from here. They would just use fully-qualified package names instead of paths.